src/pages/resultsView/ResultsViewPage.tsx

```diff
@@ -642,84 +642,94 @@ export default class ResultsViewPage extends React.Component<
                     )}
-                    {this.resultsViewPageStore.studies.isComplete && (
-                        <>
-                            <Helmet>
-                                <title>
-                                    {buildResultsViewPageTitle(
-                                        this.resultsViewPageStore
-                                            .hugoGeneSymbols,
-                                        this.resultsViewPageStore.studies.result
-                                    )}
-                                </title>
-                            </Helmet>
-                            <div>
-                                <div className={'headBlock'}>
-                                    <QuerySummary
-                                        routingStore={this.props.routing}
-                                        store={this.resultsViewPageStore}
-                                        onToggleQueryFormVisibility={visible => {
-                                            runInAction(() => {
-                                                this.showTabs = visible;
-                                                this.showOQLEditor = false;
-                                            });
-                                        }}
-                                        onToggleOQLEditUIVisibility={
-                                            this.toggleOQLEditor
-                                        }
-                                    />
-
-                                    {this.showOQLEditor && (
-                                        <div className={'quick_oql_edit'}>
-                                            <OQLTextArea
-                                                validateInputGeneQuery={true}
-                                                inputGeneQuery={
-                                                    this.quickOQLQueryStore!
-                                                        .geneQuery
-                                                }
-                                                callback={(...args) => {
-                                                    this.quickOQLQueryStore!.geneQuery =
-                                                        args[2];
-                                                }}
-                                                location={GeneBoxType.DEFAULT}
-                                                submitButton={
-                                                    this.quickOQLSubmitButton
-                                                }
-                                                error={
-                                                    this.quickOQLQueryStore!
-                                                        .submitError
+                    {this.resultsViewPageStore.studies.isComplete &&
+                        this.resultsViewPageStore.customDriverAnnotationReport
```

Comment:
could you please summarize the changes you made here? prettier makes it hard to process whats happening

Reply:
Here I make sure that the customDriverAnnotation report is referenced before the component is loaded. This way MobX will resolve the value. Before this was only happening when opening the RestultsViewSettings menu which caused a 1) rerendering of Oncoprint and 2) incorrectly represented annotations in oncoprint before the menu was opened.

Reply:
This code was removed because it turned out to be unnecessary after all.